lib/exq_scheduler/scheduler/server.ex

```diff
@@ -41,26 +43,49 @@ defmodule ExqScheduler.Scheduler.Server do
 
   def init(env) do
     storage_opts = Storage.build_opts(env)
-
-    _ = Storage.load_schedules_config(storage_opts, env)
+    schedules = Storage.load_schedules_config(env)
 
     state = %State{
-      schedules: Storage.get_schedules(storage_opts),
+      schedules: schedules,
       storage_opts: storage_opts,
       server_opts: build_opts(env),
-      env: env
+      env: env,
+      start_time: Time.now()
     }
 
-    Enum.filter(state.schedules, &Storage.is_schedule_enabled?(storage_opts, &1))
-    |> Storage.persist_schedule_times(state.storage_opts)
-
-    next_tick(self(), 0)
+    update_initial_schedule_times(:first, state)
```

Comment:
this call leads to a network call, which is not desirable inside `init`. The supervisor will wait till the init returns, so it's better to avoid any kind of IO inside the `init` callback. Just send a message `:first` and handle like we have to do in other place.

Reply:
ok, I'll change